### PR TITLE
Update longer quickstart guide to make steps clearer

### PR DESCRIPTION
**Pull request by katheris**
### Type of change

_Select the type of your PR_

- Documentation

### Description

Update longer quickstart guide to make steps clearer.

* Add links to the Apache Kafka and Kubernetes Operators docs to intro and overview sections.
* Add note that minikube user has required permissions already.
* Add step to wait for Strimzi cluster operator to be ready.
* Add delete steps.

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [ ] Supply screenshots for visual changes, such as Grafana dashboards

### CODE REVIEW

**Review by scholzj — APPROVED** [review]

File: documentation/modules/overview/con-overview-components-operators.adoc [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 [id="key-features-operators_{context}"]
 = Operators
-Strimzi provides Operators for managing a Kafka cluster running within a Kubernetes cluster.
+Strimzi provides link:https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[Operators^] for managing a Kafka cluster running within a Kubernetes cluster.
```
Comment (PaulRMellor) [inline documentation/modules/overview/con-overview-components-operators.adoc:7]:
```suggestion
Strimzi provides link:https://kubernetes.io/docs/concepts/extend-kubernetes/operator/[operators^] for managing a Kafka cluster running within a Kubernetes cluster.
```

File: documentation/modules/quickstart/proc-creating-kafka-cluster.adoc [hunk 2]
```diff
@@ -17,6 +17,7 @@ When you create a cluster, the Cluster Operator you deployed when installing Str
 .Procedure
 
 . Log in to the Kubernetes cluster as a non-privileged user.
+If using Minikube you can continue with the same user.
```
Comment (PaulRMellor) [inline documentation/modules/quickstart/proc-creating-kafka-cluster.adoc:20]:
```suggestion
If using Minikube, you can continue with the same user.
```

File: documentation/modules/quickstart/proc-installing-product.adoc [hunk 3]
```diff
@@ -12,11 +12,17 @@ Use namespaces to separate functions.
 
 .Prerequisites
 
-* Installation requires a Kubernetes account with cluster admin credentials.
+* Installation requires a Kubernetes account with privileges to install Custom Resource Definitions and Role Bindings.
```
Comment (PaulRMellor) [inline documentation/modules/quickstart/proc-installing-product.adoc:15]:
```suggestion
* Installation requires a Kubernetes account with privileges to install Custom Resource Definitions and role bindings.
```

File: documentation/modules/quickstart/proc-installing-product.adoc [hunk 4]
```diff
@@ -95,9 +104,23 @@ kubectl create -f install/cluster-operator/031-RoleBinding-strimzi-cluster-opera
...
+----
+
+. Follow the deployment of the Cluster Operator until it is marked as `Ready`.
++
+[source, shell, subs=+quotes ]
+----
+kubectl get pods -n kafka --watch
+----
++
+You can also follow the Operator's logs:
```
Comment (PaulRMellor) [inline documentation/modules/quickstart/proc-installing-product.adoc:121]:
```suggestion
You can also follow the operator's logs:
```

File: documentation/modules/quickstart/proc-uninstalling-product.adoc [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+.Procedure
+
+. To remove the Kafka cluster and associated topics, remove all Strimzi Custom Resources in the `my-kafka-project` namespace.
++
+[source, shell, subs=+quotes ]
+----
+kubectl -n my-kafka-project delete $(kubectl get strimzi -o name -n my-kafka-project)
+----
+
+. Once the Kafka cluster has been removed, delete the CRDs, role-based access control (RBAC) resources and the deployment for the Cluster Operator.
```
Comment (PaulRMellor) [inline documentation/modules/quickstart/proc-uninstalling-product.adoc:19]:
```suggestion
. When the Kafka cluster has been removed, delete the CRDs, role-based access control (RBAC) resources and the deployment for the Cluster Operator.
```

**Review by PaulRMellor — COMMENTED** [review]
Few minor suggestions. LGTM :+1:

**Review by PaulRMellor — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]

**Comment by katheris** [comment]
Thanks for the reviews, I've applied all the suggested changes

**Pull request merged**